docs/content.zh/docs/ops/metrics.md

```diff
@@ -1581,6 +1581,11 @@ Note that the metrics are only available via reporters.
       <td>The number of failed materializations.</td>
       <td>Counter</td>
     </tr>
+    <tr>
+      <td>lastMaterializationDuration</td>
```

Comment:
A minor suggestion: the name could be consistent with the metric of the size of materialization. like "lastDurationOfMaterialization" ?

Reply:
Thanks @masteryhx , good suggestion.